### ISSUE TITLE
Redundant ignore_changes element on aws_db_instance

### ISSUE THREAD

**Issue by jdgoodall1** [terraform-aws-modules/terraform-aws-rds#478]
## Description

Please provide a clear and concise description of the issue you are encountering, and a reproduction of your configuration (see the `examples/*` directory for references that you can copy+paste and tailor to match your configs if you are unable to copy your exact configuration). The reproduction MUST be executable by running `terraform init && terraform apply` without any further changes.

If your request is for a new feature, please use the `Feature request` template.

- [X] ✋ I have searched the open/closed issues and my issue is not listed.

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

- Module version [Required]: 5.4.0

- Terraform version:
1.3.0 on darwin_arm64
- Provider version(s):
+ provider registry.terraform.io/hashicorp/aws v4.53.0
+ provider registry.terraform.io/hashicorp/random v3.4.3

## Reproduction Code [Required]
```
module "rds" {
  source = "terraform-aws-modules/rds/aws"
  identifier = "<redacted>"
  engine = "postgres"
  engine_version = "13.7"
  instance_class = var.instance_class
  auto_minor_version_upgrade = true
  storage_encrypted = true
  storage_type = "gp3"
  iops = var.iops
  allocated_storage = var.storage
  max_allocated_storage = var.max_storage 
  db_name = "<redacted>"
  username = "<redacted>"
  port = "5432"
  create_random_password = true 
  multi_az = var.env == "prod" ? true : false
  maintenance_window = "<redacted>"
  backup_window = "<redacted>"
  create_db_subnet_group = true
  subnet_ids = data.aws_subnets.current.ids
  create_db_parameter_group = false 
  create_db_option_group = false 
  deletion_protection = var.env == "prod" ? true : false

}
```

1. terraform init
2. terraform plan

## Expected behavior
Plan executed without redundant ignore_changes warning

## Actual behavior
Plan executed, but displays redundant ingore_changes warning

### Terminal Output Screenshot(s)

![image](https://user-images.githubusercontent.com/26899736/217561286-56b592b5-f18f-4e9f-b428-7fdd4910f00c.png)

**Comment by jdgoodall1** [comment]
If I get the time I might raise a PR for this, but wanted to highlight in case it was already known for the next release.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 5.4.1 :tada:
